### DISCUSSION TITLE
What are four days of the week that start with "T"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1136] [Galaxy Trivia] [answered]
**Bonus:** Some months have 31 days, others have 30 days, but how many have 28 days?

**Comment by peterbaccaro** [comment]
The four days that start with "T" are **Tuesday**, **Thursday**, **today**, and **tomorrow**.
All the **12 months** have 28 days.

**Comment by CelioAmaral** [comment, marked as answer]
Four days which start with a letter "T" within the week are:

1. A week contains 7 days which is equivalent to 7x24 hours that is, 168 hours.

2. Week was not formed based on astronomy.

3. The story behind the 7 days of the week is that God worked hard for the six days and only rested on the seventh day. He took no gap in the middle of the week.

4. The names of the days in a week are named after the planets of Mercury, Saturn, Mars, Venus, and Jupiter. It was mainly done by the Babylonians.

5. Days of a week that start with the letter 'T' are listed below-

a. Tuesday

b. Thursday

c. Today

d. Tomorrow

All the 12 months have 28 days. :dart:

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Bronze 🥉 Galaxy Brain](https://github.com/CelioAmaral?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Silver 🥈 badge next by answering 16 questions. Don't forget to post your own questions too! 😊🎉

**Comment by tomerf** [comment]
Of the seven days of the week, only Tuesday and Thursday start with the letter T. This only refers to their absolute names, if you look for four days, Today and Tomorrow also start with the letter T and are relative to the current day, and are in the current week except when it's the last day of the week (Saturday/Sunday), then Tomorrow is in the next week.

Depends on how you phrase the question, only one month has **exactly** 28 days, this is February and only when it isn't a leap year, then it has 29 days. Assuming this is a trick question, all 12 months have **at least** 28 days but then the question is not worded correctly, as "_others_ have 30 days" is not correct, the months that has 31 days also have 30 days, so it should be "some/most have 30 days".

**Comment by thuongtruong1009** [comment]
Four days of the week start with the letter "T," and they are:

1. Tuesday
2. Thursday
3. Today (although not commonly used as a standalone day, it does start with "T").
4. Tomorrow (similar to "Today," it's not typically considered a standalone day, but it also starts with "T").

All months have at least 28 days. However, the phrasing of the classic riddle is a bit tricky. When you ask, "How many months have 28 days?" the answer is that all 12 months have at least 28 days. February has 28 days in common years and 29 days in leap years, while all other months have 30 or 31 days. So, technically, every month has 28 days or more.
